src/test/java/com/scottlogic/SortingTests/AuthorSort.java

```diff
@@ -14,7 +14,7 @@
 import java.util.List;
 
 
-public class AuthorSortingTests {
+public class AuthorSort {
 
     SortContext sortAuthor = new SortContext(new SortAuthor());
```

Comment:
The same thing about wrappers apply here.

Reply:
Fixed